content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
```

Comment:
Please add pros, cons, why we provide images, which images are provided and where to download them (we'll provide links in the downloads page)